features/polyExplorer/src/screens/companyDetails/companyDetails.jsx

```diff
@@ -238,7 +238,7 @@ const CompanyDetails = ({ company, onOpenRegionInfo, onOpenExploration }) => {
                                         (company.description?.value || {})[
                                             i18n.language
                                         ] || ""
-                                    ).replace("\n", "<br/><br/>") ||
+                                    ).replaceAll("\n", "<br/><br/>") ||
```

Comment:
So, that'd need some testing - I don't see how this change solves any security issue, but it does change _semantics_. Previously, we would only replace the first new line in the translations with the corresponding HTML - now it's all of them. I would _suppose_ the latter behaviour is the desired one, and I would also suppose it doesn't make a difference with the strings we currently have, but still, worth checking.

Reply:
Well, I can see how, generically, unsanitized input could inject some code. In this case it's company details, which comes from a repo that we control, but that's not something the code scanning is aware of. Also, it's just displayed, having subsequent lines wrap up instead of one on top of the other as intended is not like a big risk. Still, taking care of the CodeQL warnings if it's not a big deal can't hurt.
OTOH, although polyExplorer is one of the few features with end-to-end testing, I don't think this is included, since the DOM would probably change.
Also, what you say it did before is precisely what the security advisory says. So the change is in line with what I think was the intention, but at any rate that's why I've requested a review from @mteresa-jimenez